how-to-use/for-developers/write-script/using-pyegi-and-pyonfx-api.md

```diff
@@ -6,7 +6,7 @@ Currently, in order to write a python script for subtitle editing, you need to i
 import Pyegi
 ```
 
-We currently support [PyonFX](https://github.com/CoffeeStraw/PyonFX) library which helps with subtitle editing. In order to receive the lines from Aegisub, you need to include this code:
+You can add [PyonFX](https://github.com/CoffeeStraw/PyonFX) library to dependencies of the `pyproject.toml` file which helps with subtitle editing. In order to receive the lines from Aegisub, you need to include this code:
```

Comment:
this is a little ambiguous. developers may take this as a suggestion and go with their library of choice which is not necessarily PyonFX

Reply:
fixed: https://github.com/DrunkSimurgh/Pyegi_Docs/pull/1/files#diff-48a6f7b5121997825ec46018587b982ac0d2393eec0872f9868c0329e1844a1dR9